test/integration/features/cmd-config.feature

```diff
@@ -213,7 +213,7 @@ user defined options which changes default behaviour of Minishift.
 
   Scenario: Checking that memory value was applied
      When executing "minishift ssh -- cat /proc/meminfo" succeeds
-     Then stdout should match "MemTotal:\s*3[4-5][0-9]{5} kB"
+     Then stdout should match "MemTotal:\s*3[4-9][0-9]{5} kB"
```

Comment:
@agajdosi , I think we agreed on `Then stdout should match "MemTotal:\s*3[0-9]{6} kB"`.

Reply:
As per comment here https://issues.jboss.org/browse/CDK-256?focusedCommentId=13537041&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13537041, `MemTotal` can start with 3 or reach to 5.
 
@amitkrout @agajdosi , I think we should not worry about `3` or `4` or `5` here. It is hypervisor behavior. 

Let's go with `Then stdout should match "MemTotal:\s*\d[0-9]{6} kB"`

Reply:
@budhrg Right, We have no control over issue which is coming from hypervisor side. I am ok with this patch

Reply:
@budhrg @amitkrout then I think we need to cover also a situation with 4[0-1]**** kbs as this sometimes happens on Windows 10.

> The pass rate is 60% with the patch (failed 2 out of 5 run). In one of the test run i can see memory increase to 41* on applied 3500mb. Seems the hypervisor is responsible for inconsistent in "MemTotal" output on same applied memory in config file.